packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -173,7 +173,8 @@ export function getLanguageHeader(request) {
 }
 
 export function getClientIpHeader(request) {
-	const clientIP = request.headers['fastly-client-ip'];
+	const clientIP =
+		request.params['_set_geoip'] || request.headers['fastly-client-ip'] || null;
```

Comment:
`request.params` refers to named wildcard matches in the `path`. `request.query` corresponds to parsed querystring params

https://hapijs.com/api#request-properties

Reply:
ooh, yep, I totally misread the docs.